src/script.js

```diff
@@ -1,19 +1,51 @@
 'use strict';
 
+// import {getTopRatedFilms} from '../src/services/api.js';
+
 function createLayout(content) {
     let header = document.createElement('header');
     let main = document.createElement('main');
     let footer = document.createElement('footer');
-    let spanFooter = document.createElement('span');
-    let aHeader = document.createElement('a');
 
-    aHeader.textContent = 'HOME';
-    main.textContent = 'hello, i am main';
-    spanFooter.textContent = 'Copyright © 2022, All Right Not Reserved';
-    spanFooter.style.paddingLeft = '10%';
+    let footerSpan = document.createElement('span');
+    let footerSecSpan = document.createElement('span');
+    let headerA = document.createElement('a');
+    let headerImg = document.createElement('img');
+
+    for (let i = 0; i < 20; i++) {
```

Comment:
@Mmarvinn  this function should only render the layout and receive content and put it in the `main` tag

Reply:
fixed